%%% TeX-master: "manuscript"
% !TeX spellcheck=en_GB

\section{Discussion}

We here comprehensively analysed the phylogenetic distribution of the three enzymes using Nam as a substrate. These are the two NAD salvage pathway enzymes NADA and NamPRT as well as the Nam-degrading enzyme NNMT. We found that after the first appearance of NNMT in Protostomia, a diversification of NAD-consuming reactions in Deuterostomia can be observed. We could explain these finding using mathematical modelling, as NNMT removes excess Nam from cells and thereby reduces product inhibition of NAD signalling enzymes. This in turn enables higher fluxes through these reactions. Thus, the diversification of NAD-consuming enzymes in mammals seems to have been enabled by the presence of NNMT.

NAD-consuming enzymes are involved in a wide variety of signalling and gene regulatory mechanisms that, due to their sensitivity to NAD$^{+}$, have the ability to translate differences in metabolic states into changes in signalling and gene regulation. As NAD concentrations are lowered by the removal of NAD precursor by NNMT, a high-affinity enzyme for recycling of Nam is required to ensure maintenance for cellular NAD levels for other metabolic processes. It is unclear, why NamPRT developed a higher affinity and not NADA. NADA may have thermodynamic or mechanistic limitations or other environmental conditions may have been responsible for the observed evolutionary trajectory.  Still, our simulations clearly show that a high affinity of NamPRT is required for high NAD consumption fluxes and NAD concentrations. It therefore seems plausible that NNMT might have been driving NamPRT evolution. Looking at the enzyme affinities of the human enzymes it furthermore appears that both NNMT and NamPRT reached an almost optimal state, as further changes in the affinity of NamPRT or NNMT would not result in much higher steady state NAD concentrations or NAD consumption fluxes, according to our simulations. In addition, the data suggest that NNMT makes both NAD concentration and NAD consumption relatively independent of other processes requiring NAD, such as cell growth.

Our findings shed also new light on the potential physiological role of NNMT, which has earlier been recognized as potential marker for some types of cancer (e.g. \cite{Okamura1998}). The major healthy tissue expressing NNMT is the liver, while no or only little expression of NNMT is observed in most other tissues \todo{ref}. Increased NNMT expression in cancer might serve to remove Nam produced by increased NAD-dependent signalling. To maintain high NAD concentrations, a simultaneous higher expression of NamPRT is required, which is what has been found in some types of cancer \cite{Bi2011,Wang2011}. NNMT is only advantageous as long as NamPRT affinity is sufficiently high. This suggests that certain types of cancer that express NNMT at a high level would be more susceptible to competitive inhibitors of NamPRT. Several of such inhibitors are currently tested in clinical studies \cite{Espindola-Netto2017} \todo{Are there more refs?}. Based on our analysis, we would suggest that it might be reasonable to screen patients before treatment, as non-NNMT expressing tumours might respond less to competitive NamPRT inhibitors and missing Nam degradation in those cancer cells would potentially lead to an accumulation of Nam that could outcompete the inhibitor. The latter aspect is unclear and requires further investigation.

Our combined phylogenetic/modelling analysis provides a potential explanation both for the co-occurrence of NADA and NamPRT in bacteria and for the loss of NADA in vertebrates. We here show that when two compartments compete for the same limited source of Nam, the compartment that contains NamPRT and NNMT has a higher steady state NAD concentration and NAD consumption rate than the compartment containing NADA. The dominant enzyme combination found in vertebrates, a high-affinity NamPRT with NNMT, seems to provide a competitive advantage. As this may also hold for mammalian-associated bacteria, particularly pathogens, we wanted to see whether pathogenic bacteria solely express NamPRT. Unfortunately, bacterial habitat information is currently far from complete and often difficult to access. We therefore manually checked bacteria that possess NamPRT and indeed found that most of them have been characterised to be pathogenic. It should be noted that the distribution of NADA and NamPRT does not follow the bacterial species tree but is rather scattered \cite{Gazzaniga2009}. This distribution hints to a functional importance of posssessing NADA or NamPRT that might be due to different environments.

A detailed analysis of sequence variances in NamPRT revealed that only deuterostomes that have NNMT but not NADA, have a sequence insertion in the N-terminal part of NamPRT that seems to enable the high affinity of the enzyme. This in turn would suggest that also the bacterial enzymes do not have a high substrate affinity. The only kinetically characterised NamPRT is from \textit{Acinetobacter baylyi} \cite{Sorci2010} and has a $K_{M}$ for Nam that is about 10\,000 times higher than the $K_{M}$ of human NamPRT, supporting our hypothesis. Other bacterial NamPRTs were shown to be functional \cite{Martin2001,Gerdes2006}, but no kinetic parameters were measured. The differences in activity and affinity, implying differences in substrate binding could potentially be exploited for the development of antibiotics. Further analysis possibly including the crystallisation of a bacterial NamPRT would be required, to see whether the bacterial NAD metabolism could be a promising target.

% Martin2001: Haemophilus ducreyi: no kinetic measurements
% Sorci2010: Acinetobacter sp.: NamPRT (Nam) Km: 0.04 mM (= 40 000 nM); kcat: 0.12/s (apparent values determined at constant 5 mM PRPP and 2 mM ATP)
% Gerdes2006: Synechocystis sp.: only activity 0.5 U/mg NamPRT for Nam
% Human, according to table S2: Km: 5 nM; kcat: 0.0077/s

In our analysis, we completely neglected the potential effects of co-substrates of the investigated pathway. Neglected co-substrates include targets of the NAD-consuming enzymes, such as acylated proteins for sirtuins, for example, or phosphoribosyl pyrophosphate (PRPP) and ATP that are required for NMN synthesis by NamPRT. Also the presence of the methyl donor \textit{S}-adenosyl methionine (SAM) and its precursor methionine that have been shown to potentially limit the effect of NNMT \cite{Ulanovskaya2013} was not considered. All of these co-substrates should be included in future analyses, but information about the \textit{in~vivo} concentration of those co-substrates is currently very limited. We anticipate improvements as more sensitive methods for metabolite measurements are currently developed \todo{ref? Or how do you know?}.

% todo: The following three paragraphs were not proofread, yet.

During our analysis we came across several problems related to the NCBI sequence database. One is sequence contamination, which is a well known problem \todo{ref?} and we therefore tried to remove all sequences of obvious bacterial origin from the analysis, based on sequence homology analysis. Another problem is incomplete genomes. Although there are tools to assess the completeness of a genome (e.g.~\cite{Simao2015}), none of them could convincingly claim to be reliable. The genomes of the common model organisms can probably be assumed to be close to complete, but there are many draft genomes in the databases whose completeness is uncertain. Even if the completeness would be known, with the high number of genomes used in this analysis, it is likely that some genes of interest were not sequenced in every genome. For our analysis, this means that scattered patterns of few missing genes may be real or stem from an incomplete genome.

The third problem are wrong annotations. We tried to avoid these, by only relying on template sequences with confirmed function, wherever possible. This problem becomes apparent by the fact that in yeast an enzyme named NNMT can be found, which is based on an initial analysis on life span extension in \textit{Saccharomyces cerevisae} \cite{Anderson2003}. The protein has, however, later been shown not function as methyltransferase for Nam but for the eukaryotic elongation factor~1A (eEF1A) giving it its new name elongation factor methyltransferase~7 (Efm7) \cite{Hamey2016}. The old name is still present in many databases, though.

Nevertheless, we have been able to comprehensively analyse the functional co-evolution of several enzymes of the NAD pathway with the appearance of NNMT seemingly initiating and driving complex alterations of the pathway such as an increase and diversification of NAD-dependent signalling, followed by an increase in NamPRT substrate affinity (schematic overview see fig.~\ref{fig:evo_events}). This again appears to be accompanied by the loss of NADA in vertebrates and the first gene duplication of NMNATs \cite{Lau2010}. We also noted that the second gene duplication of NMNATs and thus the further compartmentalisation of NAD metabolism is co-occurring with a site-specific positive selection event in NNMT (unpublished results). To our knowledge, this is the first study that comprehensively demonstrates a interdependency of enzyme evolution based on functional interaction of several enzymes of a pathway. While it is of course well know and described for directly interaction proteins \todo{Not sure that is true, one that might come close is the following work PMID:26446903 \cite{Figliuzzi2016}. Please check and potentially revise statement}.
