%%% TeX-master: "manuscript"

\section{Discussion}

\subsection{Evolutionary and physiological role of NNMT}

The dynamic interactions among metabolism, signal transduction, and gene regulation is still not very well understood. New approaches are required to disentangle the underlying network, helping us to understand how alterations affect human physiology. We here combine phylogenetic analysis and mathematical modelling of a central metabolic pathway supported by experimental analysis to reveal the likely evolutionary development of the NAD biosynthesis and consumption pathway and the physiological role of the methyltransferase NNMT.

We show that the vitamin B3-degrading enzyme NNMT plays a vital role for the diversification of NAD-dependent signalling reactions and potentially also for NAD homeostasis. Due to the fact that NamPRT has a very high affinity for Nam whereas NNMT has a rather low affinity for it, NNMT is able to remove excess Nam that would lead to inhibition of NAD-dependent signalling reactions while maintaining NAD concentrations and even increasing NAD consumption fluxes.

The affinities measured for the human enzymes seem to be optimal and further increase in NAD consumption rates might only enabled by compartmentalisation of NAD biosynthesis.


\subsection{General applicability}

It is of course impossible to extend our conclusions to other metabolic processes without analysing them in more detail, but we have noted that the metabolism of other vitamins, such as pyridoxal (vitamin B6) metabolism, also contains vitamin degrading enzymes \todo[author=Mathias]{source}, that might thus have similar roles. Degrading enzymes do in general not receive the same attention as biosynthetic enzymes, reflected by the observation that NNMT has only recently been analysed in more detail, besides the much earlier recognition of NNMT as potential marker for some types of cancer (e.g. Ref?) and routine clinical measurements of urine methyl-Nam concentrations in the context of different diseases (e.g. Ref?). Our analysis show, however, that degrading enzymes do not solely modify substrates for better excretion, but can play a vital role both in human physiology as well as in the evolutionary development of biological processes.


\subsection{Limitations of our analysis and outlook}

Among others, we neglected the potential effects of the methyl donor S-adenosyl-methionin (SAM) and its precursor methionine in our analysis, although it is most likely contributing yet another regulatory level for Nam availability and thus an additional interaction point between gene regulation and metabolism. It has been shown earlier that in turn, NNMT expression influences protein methylation dependent on methionine availability \cite{Ulanovskaya2013}. One of the challenges in the analysis of this aspect is the availability of in vivo concentration measurements for SAM and the large amount of reactions using it as substrate. The same holds for the analysis of enzyme acetylation versus deacetylation by sirtuins that would enable prediction about enzyme activation or histone state if available.


\section{Experimental Procedure}

\subsection{Dynamic modelling}

Kinetic parameters (substrate affinity (Km) and turnover rates (kcat), substrate and product inhibitions) were retrieved from the enzyme database BRENDA and additionally evaluated by checking the original literature especially with respect to measurement conditions. Parameter values from mammals were used if available. For enzymes not present in mammals, values from yeast were used. The full list of kinetic parameters including reference to original literature can be found in Supplementary table 1. For NMNAT, the previously developed rate law for substrate competition was used \cite{Schauble2013}. Despite these modifications, Henri-Michaelis-Menten kinetics were used for all reactions except the import and efflux of Nam, which were simulated using constant flux and mass action kinetics, respectively. All simulations were performed using the steady state calculation and parameter scan options provided by COPASI 4.22 \cite{Hoops2006}. The model will be available at the Biomodels database accession number \todo[inline]{xxx}. Related figures were generated using gnuplot\todo[author=Mathias]{Version?}.


\subsection{Phylogenetic Analysis}

NADA, NamPRT, and NNMT enzymes or enzyme candidates were identified using Blastp with known enzymes against the NCBI non-redundant protein sequence database (nr). A list of functionally verified enzymes used as templates is given in supplementary table 2. This table also includes the length cut-offs for identified enzymes. The e-value cut-off was 1e-30 for all enzymes. Blastp parameters were set to yield maximum 20,000 target sequences, using the BLOSUM62 matrix with a word size of 6 and gap opening and extension costs of 11 and 1, respectively. Low-complexity filtering was disabled. Obvious sequence contaminations were removed by manual inspection of the results. The taxonomy IDs of the species for each enzyme was derived from the accession2taxonomy database provided by NCBI. Scripts for creating, analysing, and visualising the phylogenetic tree were written in Python, using the ETE3 toolkit (Huerta-Cepas, 2010).


\subsection{Generation of eukaryotic expression vectors encoding C-terminally FLAG-tagged NamPT proteins}

The open reading frame (ORF) encoding human NamPRT was inserted into pFLAG-CMV-5a (Merck - Sigma Aldrich) via EcoRI/BamHI sites. Using a PCR-based approach, this vector provided the basis for the generation of a plasmid encoding a NamPRT deletion mutant lacking amino acid residues 42-51 (NamPRT-$\Delta$42-51). The sequences of the inserted ORFs were verified by DNA sequence analysis.


\subsection{Transient transfection, immunocytochemstry and confocal laser scanning microscopy}

HeLa S3 cells cultivated in Ham’s F12 medium supplemented with 10\% (v/v) FCS, 2 mM L-glutamine, and penicillin/streptomycin, were seeded on cover slips in a 24 well plate. After one day, cells were transfected using Effectene transfection reagent (Qiagen) according to the manufacturer’s recommendations. Cells were fixed with 4\% paraformaldehyde in PBS 24 hours post transfection, permeabilised (0.5\% (v/v) Triton X-100 in PBS) and blocked for one hour with complete culture medium. After overnight incubation with primary FLAG-antibody (mouse M2, Sigma-Aldrich) diluted 1:2500 in complete medium, cells were washed and incubated for one hour with secondary AlexaFluor 594-conjugated goat anti mouse antibody (ThermoFisher, Invitrogen) diluted 1:1000 in complete culture medium. Nuclei were stained with DAPI and the cells washed. The cover slips were mounted on microscope slides using ProLong Gold (ThermoFisher, Invitrogen). Confocal laser scan imaging of cells was performed using a Leica TCS SP8 STED 3x microscope equipped with a 100x oil immersion objective (numerical aperture 1.4).