%%% TeX-master: "manuscript"
% !TeX spellcheck=en_GB

\section{Discussion}

We here analysed the phylogenetic distribution of the three enzymes using Nam as a substrate, the two NAd-salvage pathway enzymes NADA and NamPRT as well as the Nam degrading enzyme NNMT. We found that after the first appearance of NNMT in Protostomia, a diversification of NAD-consuming reactions in Deuterostomia can be observed. Using mathematical modelling, we can explain these finding, as NNMT removes excess of Nam in cells and by that reduces product inhibition of NAD-signalling enzymes, enabeling higher fluxes through these reactions. This in turn seem to have enabled a diversification and NAD-consuming enzymes in mammals are involved in a wide variety of signalling and gene regulatory mechanisms that due to there sensitivity to NAD+, have the ability to translate differences in metabolic  states into changes in signalling and gene regulation. As NAD concentrations are consequently lowered, a high affinity enzyme for recycling of Nam is required to ensure maintenance for cellular NAD-levels for other metabolic processes. Whether it was by chance or whether an this is due to mechanistic limitations that NamPRT developed a high affinity and not NADA, or whether other environmental conditions have been responsible is impossible to deduce.  But our simulations clearly show that a high affinity of NamPRT is favourable and that it therefore appears that NNMT might have been driving NamPRT evolution. Looking at the enzyme affinities of the human enzymes it furthermore appears  that both NNMT and NamPRT reached an near optimal state, as further increase of the affinity of NamPRT and decrease of the affinity of NNMT would not result in much higher steady state NAD-concentrations or NAD-consumption fluxes, according to our simulations.  It furthermore appears that NNMT makes both NAD concentration and NAD consumption relatively independent of other processes requiring NAD, such as cell growth. In addition. 

Our findings shed also new light on the potential physiological role of NNMT, which has earlier been recognized as potential marker for some types of cancer (e.g. \cite{Okamura1998}). It might well be that increased NNMT expression in cancer might serve to remove Nam produced by increased NAD-dependent signalling. To maintain high NAD-concentration a simultaneous overexpression of NamPRT is required, which is what has been found in some types of cancer (ref).  As NNMT does, however, provide an advantage as long as NamPRT affinity is sufficiently high, this might provide an useful avenue for treatment, especially for types of cancer that express NNMT at a high lever. These would be more susceptible to competitive inhibitors of NamPRT, and several of those inhibitors are currently tested in clinical studies. The major healthy tissue expressing NNMT is the liver, while most other tissues do not express NNMT or only at vary low levels.  Based on our analysis we would furthermore suggest, that it might be reasonable to screen patients before treatment, as non NNMT expressing tumours might respond less to competitive NamPRT inhibitors. This is furthermore relevant, as the missing degradation in those cells would potentially lead to an accumulating of Nam that could outcompete the inhibitor in those cells. But these aspects require further investigation.

Our combine phylogenetic-modelling analysis furthermore provides a potential explanation both for the co-occurance of NADA and NamPRT in bacteria, but even more strongly for the loss of NADA in vertebrates. We here show that when  simulating compartments that share the same limited Nam-source, the compartment  that contains NamPRT and NNMT has a higher steady state NAD concentration and NAD consumption rate as the compartment containing NADA. The combination of a high affinity NamPRT with NNMT which is  the dominant enzyme combination found in vertebrates seems to provide a competitive advantages. As this may also hold for mammalian associated bacteria, especially pathogens, we wanted to see whether pathogenic bacteria solely express NamPRT. Unfortunately, bacterial habitat information is currently not complete and often difficult to access especially at the NCBI database that was used for the analysis provided here, as it is the most comprehensive sequence database to date. We therefore manually looked through the bacteria found to encode for NamPRT and indeed found that many but not all of them have been characterised to be pathogenic. It should be noted that the distribution of NADA and NamPRT does not follow the bacterial species tree\todo{insert ref}. 

A detailed analysis of sequence variances in NamPRT revealed that only Deuterostomia that have NNMT but not NADA, have a sequence insertion in the N-terminal part of NamPRT that according to our experimental analysis seems to enable the high affinity of the enzyme. This in turn would suggest that also the bacterial enzymes doe not have a high substrate affinity and although the one bacterial NamPRT has been proven to be enzymatically active, the affinity of a bacterial enzyme has not been measured so far. The differences in activity and affinity, implying differences in substrate binding could potentially be exploited for the development of  antibiotics. But further analysis as well as potentially the crystallisation of a bacterial enzyme would be required, to see whether this could be a  approach. 

In our analysis, we so far completely neglected the potential effects of cosubstrates of the pathway investigated such as the presence of targets of the NAD-consuming enzymes, such as acetylated proteins for sirtuins for example, or the requirement of PRPP\todo{insert full name and but PRPP in brackets} and ATP that are rrequired for NMN synthesis by NamPRT  or the presence of methyl donor \textit{S}-adenosyl methionine (SAM) and its precursor methionine that have been shown to potentially limit the effect of NNMT\cite{PMID:23455543 }. These should be included in future analysis. The challenge is, however, that currently information about the in vivo concentration of those co-substrates are very limited. We assume that this will improve in the future as new, more sensitive, methods for metabolite measurements are currently developed.

During our analysis we came across several problems related to current sequence database. One is sequence contamination, which is a well known problem\todo{insert ref} and we therefore tried to remove all sequences of obvious bacterial origin from the analysis, based on sequence homology analysis. Another problem are incomplete genomes .... \todo{Mathias B. please insert here the paragraph you have written for your thesis--- you should probably write it in other words}

The third problem are wrong annotations. We tried to avoid these, by, wherever possible, only relying on template sequences with confirmed function. This problem becomes apparent by the fact that in yeast an enzyme named NNMT can be found, which is based on an initial analysis on life span extension in Saccharomyces cerevisae\todo{insert ref} a sequence similarity that is restricted to the SAM-dependent methyl transferase domain.  The protein has, however, later been shown not function as Nam or small molecule methyl transferase but as ...\todo{Please complete incl. refs.}

Nevertheless, we have been able to comprehensively analyse co-evolution of several enzymes of the NAD pathway with the appearance of NNMT seemingly initiating and driving complex alterations of the pathway such as an increase and diversification of NAD-dependent signalling, followed by an increase in NamPRT substrate affinity (schematic overview see fig.~\ref{fig:evo_events}). This again appears to be accompanied by the loss of NADA in vertebrates and the first gene duplication of NMNATs \cite{Lau2010}. We also noted that the second gene duplication of NMNATs and thus the further compartmentalisation of NAD metabolism is co-occurring with a site-specific positive selection event in NNMT (unpublished results). To our knowledge, this is the first study that comprehensively demonstrates a interdependency of enzyme evolution based on functional interaction of several enzymes of a pathway. While it is of course well know and described for directly interaction proteins \todo{Not sure that is true, one that might come close is the following work https://academic.oup.com/mbe/article/33/1/268/2579532.....please check and potentially revise statement}.






