%%% TeX-master: "manuscript"
% !TeX spellcheck=en_GB

\section{Discussion}

We here comprehensively analysed the phylogenetic distribution of the three enzymes using Nam as a substrate that are the two NAD salvage pathway enzymes NADA and NamPRT as well as the Nam-degrading enzyme NNMT. We found that after the first appearance of NNMT in Protostomia, a diversification of NAD-consuming reactions in Deuterostomia can be observed. By using mathematical modelling, we could explain these finding, as NNMT removes excess Nam from cells and thereby reduces product inhibition of NAD signalling enzymes. This in turn enables higher fluxes through these reactions. Thus, the diversification of NAD-consuming enzymes in mammals seems to have been enabled by the presence of NNMT. NAD-consuming enzymes are involved in a wide variety of signalling and gene regulatory mechanisms that due to their sensitivity to NAD$^{+}$, have the ability to translate differences in metabolic states into changes in signalling and gene regulation. As NAD concentrations are lowered by the removal of NAD precursor by NNMT, a high-affinity enzyme for recycling of Nam is required to ensure maintenance for cellular NAD levels for other metabolic processes. Whether it was by chance or whether this is due to thermodynamic or mechanistic limitations that NamPRT developed a high affinity and not NADA, or whether other environmental conditions have been responsible for the evolutionary trajectory observed, is impossible to deduce based on the chosen approach. But our simulations clearly show that a high affinity of NamPRT is required for high NAD consumption fluxes and NAD concentrations and that it therefore appears that NNMT might have been driving NamPRT evolution. Looking at the enzyme affinities of the human enzymes it furthermore appears that both NNMT and NamPRT reached an near optimal state, as further increase of the affinity of NamPRT and decrease of the affinity of NNMT would not result in much higher steady state NAD concentrations or NAD consumption fluxes, according to our simulations. It furthermore appears that NNMT makes both NAD concentration and NAD consumption relatively independent of other processes requiring NAD, such as cell growth.

Our findings shed also new light on the potential physiological role of NNMT, which has earlier been recognized as potential marker for some types of cancer (e.g. \cite{Okamura1998}). It might well be that increased NNMT expression in cancer might serve to remove Nam produced by increased NAD-dependent signalling. To maintain high NAD concentrations a simultaneous overexpression of NamPRT is required, which is what has been found in some types of cancer \cite{Bi2011,Wang2011}. As NNMT does, however, provide an advantage as long as NamPRT affinity is sufficiently high, this might provide an useful avenue for treatment, especially for types of cancer that express NNMT at a high level. These would be more susceptible to competitive inhibitors of NamPRT, and several of those inhibitors are currently tested in clinical studies \todo{ref. Maybe PMID:28733523 \cite{Espindola-Netto2017}?}. The major healthy tissue expressing NNMT is the liver, while most other tissues do not express NNMT or only at very low levels. Based on our analysis, we would furthermore suggest that it might be reasonable to screen patients before treatment, as non-NNMT expressing tumours might respond less to competitive NamPRT inhibitors. This is furthermore relevant, as the missing degradation in those cells would potentially lead to an accumulating of Nam that could outcompete the inhibitor in those cells. But these aspects require further investigation.

Our combined phylogenetic/modelling analysis furthermore provides a potential explanation both for the co-occurrence of NADA and NamPRT in bacteria, but even more strongly for the loss of NADA in vertebrates. We here show that when simulating compartments that share the same limited source of Nam, the compartment that contains NamPRT and NNMT has a higher steady state NAD concentration and NAD consumption rate as the compartment containing NADA. The combination of a high affinity NamPRT with NNMT, which is the dominant enzyme combination found in vertebrates, seems to provide a competitive advantage. As this may also hold for mammalian-associated bacteria, particularly pathogens, we wanted to see whether pathogenic bacteria solely express NamPRT. Unfortunately, bacterial habitat information is currently far from complete and often difficult to access especially at the NCBI database that was used for the analysis provided here, as it is the most comprehensive sequence database to date. We therefore manually looked through the bacteria found to encode for NamPRT and indeed found that many but not all of them have been characterised to be pathogenic. It should be noted that the distribution of NADA and NamPRT does not follow the bacterial species tree\todo{ref. Maybe PMID:19721089 \cite{Gazzaniga2009}?}.

A detailed analysis of sequence variances in NamPRT revealed that only Deuterostomia that have NNMT but not NADA, have a sequence insertion in the N-terminal part of NamPRT that, according to our experimental analysis, seems to enable the high affinity of the enzyme. This in turn would suggest that also the bacterial enzymes do not have a high substrate affinity and although the one bacterial NamPRT has been proven to be enzymatically active, the affinity of a bacterial enzyme has not been measured so far \todo{There were more NamPRTs found and (some of) their kinetic constants were measured. Please have a look at e.g. PMIDs:11157928, 20926389, 16585762 \cite{Martin2001,Sorci2010,Gerdes2006}}. The differences in activity and affinity, implying differences in substrate binding could potentially be exploited for the development of antibiotics. But further analysis as well as potentially the crystallisation of a bacterial enzyme would be required, to see whether this could be a promising approach.

In our analysis, we so far completely neglected the potential effects of co-substrates of the pathway investigated such as the presence of targets of the NAD-consuming enzymes, such as acetylated proteins for sirtuins for example, or the requirement of phosphoribosyl pyrophosphate (PRPP) and ATP that are required for NMN synthesis by NamPRT or the presence of the methyl donor \textit{S}-adenosyl methionine (SAM) and its precursor methionine that have been shown to potentially limit the effect of NNMT \cite{Ulanovskaya2013}. These should be included in future analysis. The challenge is, however, that currently information about the \textit{in~vivo} concentration of those co-substrates is very limited. We assume that this will improve in the future as new, more sensitive, methods for metabolite measurements are currently developed \todo{ref?}.

During our analysis we came across several problems related to the NCBI sequence database. One is sequence contamination, which is a well known problem \todo{ref?} and we therefore tried to remove all sequences of obvious bacterial origin from the analysis, based on sequence homology analysis. Another problem is incomplete genomes. Although there are tools to assess the completeness of a genome (e.g.~\cite{Simao2015}), none of them could convincingly claim to be reliable. The genomes of the common model organisms can probably be assumed to be close to complete, but there are many draft genomes in the databases whose completeness is uncertain. Even if the completeness would be known, with the high number of genomes used in this analysis, it is likely that some genes of interest were not sequenced in every genome. For our analysis, this means that scattered patterns of few missing genes may be real or stem from an incomplete genome.

The third problem are wrong annotations. We tried to avoid these, by only relying on template sequences with confirmed function, wherever possible. This problem becomes apparent by the fact that in yeast an enzyme named NNMT can be found, which is based on an initial analysis on life span extension in \textit{Saccharomyces cerevisae} \cite{Anderson2003}. The protein has, however, later been shown not function as methyltransferase for Nam but for the eukaryotic elongation factor~1A (eEF1A) giving it its new name elongation factor methyltransferase~7 (Efm7) \cite{Hamey2016}. The old name is still present in many databases, though.

Nevertheless, we have been able to comprehensively analyse the functional co-evolution of several enzymes of the NAD pathway with the appearance of NNMT seemingly initiating and driving complex alterations of the pathway such as an increase and diversification of NAD-dependent signalling, followed by an increase in NamPRT substrate affinity (schematic overview see fig.~\ref{fig:evo_events}). This again appears to be accompanied by the loss of NADA in vertebrates and the first gene duplication of NMNATs \cite{Lau2010}. We also noted that the second gene duplication of NMNATs and thus the further compartmentalisation of NAD metabolism is co-occurring with a site-specific positive selection event in NNMT (unpublished results). To our knowledge, this is the first study that comprehensively demonstrates a interdependency of enzyme evolution based on functional interaction of several enzymes of a pathway. While it is of course well know and described for directly interaction proteins \todo{Not sure that is true, one that might come close is the following work PMID:26446903 \cite{Figliuzzi2016}. Please check and potentially revise statement}.
