% !TeX spellcheck=en_GB

\RequirePackage[l2tabu, orthodox]{nag}
\documentclass[paper=a4, 12pt]{scrartcl}

\usepackage{graphicx}
\usepackage[font=footnotesize]{caption}
\usepackage[utf8]{inputenc}
\usepackage[colorlinks=true,
	linkcolor=black,
	citecolor=black,
	filecolor=black,
	urlcolor=black]{hyperref}
\usepackage{textcomp}
\usepackage[left=1cm,right=5cm,top=2cm,bottom=2cm,nohead,nofoot]{geometry}
\usepackage[textsize=scriptsize]{todonotes}

\setlength{\marginparwidth}{4cm}

\frenchspacing
\graphicspath{ {../} }

\begin{document}


\noindent
{\huge\sffamily\bfseries NamPRT and NNMT – key drivers of NAD-dependent signalling \par}

\vspace{15mm}

\noindent
\textit{Alternative title:} Functional interdependencies as driving force for the evolution of NAD-dependent signalling and biosynthesis

\vspace{5mm}

\noindent
Mathias Bockwoldt\textsuperscript{1}, Dorothée Houry\textsuperscript{2}, Marc Niere\textsuperscript{2}, Toni I. Gossmann\textsuperscript{3}, Mathias Ziegler\textsuperscript{2}, and Ines Heiland\textsuperscript{1,§}

\vspace{1cm}

\noindent
\textsuperscript{1}Department of Arctic and Marine Biology, UiT The Arctic University of Norway, Biologibygget, Framstredet 39, 9017 Tromsø, Norway

\noindent
\textsuperscript{2}Department of Molecular Biology, University of Bergen, Thormøhlensgt. 55, 5020 Bergen, Norway

\noindent
\textsuperscript{3}Department of Animal and Plant Sciences, Western Bank, University of Sheffield, Sheffield, S10 2TN, United Kingdom

\noindent
§ Corresponding author: ines.heiland@uit.no


\section*{Summary}

NAD is best known as cofactor in redox reactions, but it is also substrate of NAD-dependent signalling reactions that consume NAD and release nicotinamide (Nam). In eukaryotes, two different Nam salvage pathways exist. Phylogenetic analysis of the pathway show that in lower organisms the initial deamidation of Nam is prevalent, whilst the direct conversion of Nam to the mononucleotide by Nam phosphoribosyltransferase (NamPRT) dominates in animals and eventually remains as the single Nam recycling route in vertebrates. Strikingly, loss of the deamidation pathway in early vertebrates is preceded by the occurrence of a new enzyme that marks Nam for excretion by methylation – nicotinamide N-methyltransferase (NNMT). The physiological relevance of NNMT is still enigmatic. Why is there an enzyme that removes Nam from recycling, seemingly not having any other physiological role? Despite the occurrence of this Nam degrading enzyme NNMT, we observe a diversification of NAD dependent signalling enzymes in vertebrates. This seems counterintuitive.

To better understand these evolutionary changes and the role of NNMT for NAD metabolism, we built a mathematical model of the pathway using published enzyme kinetics data. Our results indicate that NNMT is required to enable high NAD consumption fluxes necessitated by the increasing diversification of NAD-dependent signalling pathways. This kinetic regulation requires a high substrate affinity of the key enzyme for Nam salvage, NamPRT. Indeed, the affinity of NamPRT to Nam has previously been measured to be in the nanomolar range. Based on our mathematical modelling approach we furthermore suggest that NNMT exerted an evolutionary pressure on NamPRT, enforcing the development of its unusually high substrate affinity. Using multiple sequence alignments, we identified a sequence insertion, first occurring in vertebrates, that parallels an experimentally verified increase in the substrate affinity of the enzyme. Additional simulations show that the deamidation pathway became obsolete owing to the high substrate affinity of NamPRT. Collectively, our results illustrate a close evolutionary relationship between NAD biosynthesis and the diversification NAD-dependent signalling pathways, potentially driven by the concomitant occurrence of a regulator of Nam salvage, NNMT.


\section*{Keywords}


\input{introduction}

\input{results}

\input{discussion}

\input{methods}

\bibliographystyle{plain}
\bibliography{library}

\input{figurelegends}

\input{supplementaries}

\section*{Acknowledgements}

We thank the Norwegian Research Council for funding (grant no. 250395/F20). We furthermore thank for the computation time provided through UNINETT Sigma2 – the National Infrastructure for High Performance Computing and Data Storage in Norway.

\end{document}
