%%% TeX-master: "manuscript"
% !TeX spellcheck=en_GB

\section{Results}

\subsection{Paradoxical evolutionary correlation between NAD-dependent signalling and precursor metabolism}

We have previously shown that NMNAT is essential for NAD biosynthesis independent of the biosynthetic precursor or pathway \cite{DeFigueiredo2011}. In addition, the Preiss-Handler pathway was found  present in virtually all organisms included in an earlier phylogenetic analysis \cite{Gossmann2012FEBS}. Thus, we here focused on the phylogenetic analysis of the two enzymes that initiate the two different NAD salvage pathways, NADA and NamPRT. The phylogenetic distribution of the two enzymes is scattered in bacteria \todo{MathiasB: please find and insert citation} and have been shown to occur together in some marine invertebrates \cite{Gossmann2012FEBS}. In addition, we included the Nam methyltransferase NNMT into our analysis, as its function and origin is unclear.

As shown in Figure~\ref{fig:phylo_distribution}A, bacteria, fungi, and plants predominantly possess NADA and only a few of them harbour NamPRT. In contrast, Metazoa predominantly lost NADA amd have NamPRT together with NNMT. NNMT seems to have arisen \textit{de~novo} in the most recent common ancestor of Ecdysozoa and Lophotrochozoa (fig.~\ref{fig:phylo_distribution}B). As we were unable to find any \textit{NNMT} gene with an e-value below xxx \todo{Mathias B. include e-value} in fungi or plants. The gene with the highest sequence similarity in \textit{S.~cerevisiae} is ... with proven function ... \todo{Mathias B: include values}.

Nematodes are the only organisms, where we observed a concomitant presence of NADA and NNMT. In deuterostomes, the only large clade that possesses only NamPRT and seems to have lost NNMT are Sauropsida, and among them especially birds. The reason why about half of the sequenced bird genomes do not seem to encode for NNMT remains unclear. The distribution of NNMT in birds is quite scattered (suppl. fig.~S2). \todo{figure not corresponding to current species trees \cite{Prum2015}} The lack of NNMT might be related to the excretion system, as the product of NNMT, methyl-Nam, is in mammals excreted with the urine. There are few metazoan species for which we could not find NamPRT or NADA, while NNMT was present. We assume that this is due to incomplete genomes in the database, as the distribution of such species is scarce and widely scattered.

In addition to the phylogenetic distribution of the two Nam salvage enzymes NADA and NamPRT, we analysed the phylogenetic diversity of enzymes catalysing NAD-dependent signalling reactions. To do so, we used the previously established classification into ten different families of NAD-consuming signalling enzymes \cite{Gossmann2012FEBS}. The detailed list of templates used for the phylogenetic analyses can be found in supplementary table~S1. The numbers shown in figure~\ref{fig:phylo_distribution}B denote the average number of NAD-dependent signalling enzyme families found in each clade. With the exception of Cnidaria and Lophotrochozoa, we find an average of three to four families in protostomes, whereas most deuterostome species have, on average, more than eight families with an increasing diversification of enzymes within some of these families \cite{Gossmann2014DNAR}.

Taken together, we found that NADA is lost in vertebrates, but strongly preserved in most other organisms, despite the higher energetic requirement of that pathway. Moreover, the selection for having both NamPRT and NNMT coincides with an increased diversification of NAD-dependent signalling. This observation seems counterintuitive, as one would expect that increased NAD-dependent signalling should be accompanied by an increase of substrate availability for NAD biosynthesis.


\subsection{Functional properties of NamPRT and NNMT have evolved to maximize NAD-dependent signalling}

To resolve this apparent contradiction, we wished to scrutinize the NAD metabolic network. Given the complexity of this network, we turned to modelling approaches and built a dynamic model of NAD metabolism based on previously reported kinetic data (for details, see experimental procedures and suppl. tab.~S2).

To be able to compare metabolic features of evolutionary quite different systems in our simulations and as we had limited information about expression levels of enzymes, we initially used equal amounts for all enzymes. As we have very few cross species kinetic data, we were furthermore mainly relying on kinetic constants found for human or yeast enzymes. Wherever possible, we did not only include substrate affinities but also known product inhibitions or inhibition by downstream metabolites. As we in addition assumed that cell growth is, besides NAD-consuming reactions, a major driving force for NAD biosynthesis, we analysed different growth rates (cell division rates) by simulating different dilution rates for all metabolites.

First, we addressed the unexpected correlation between the selection for co-occurrence of NamPRT and NNMT and an increase in the number of NAD-consuming enzyme families. We calculated steady state NAD concentrations and NAD consumption rates by simulating NAD biosynthesis proceeding via NamPRT in the presence or absence of NNMT. To achieve free NAD concentration in the range reported in the literature and due to the very low turnover of NamPRT, we had to use tenfold higher NamPRT levels compared to other enzymes and adjusted the amount of NMNAT to the same level to avoid that the NAD synthesis rates are limited by this enzyme. As shown in figure~\ref{fig:NNMT_NAD_flux}, the presence of NNMT enables higher NAD consumption rates (fig.~\ref{fig:NNMT_NAD_flux}A) while reducing the steady state concentration of NAD (fig.~\ref{fig:NNMT_NAD_flux}B). The decline in NAD concentration can be compensated by a higher expression of NamPRT, further increasing NAD consumption flux (dashed lines in fig.~\ref{fig:NNMT_NAD_flux}A and~B).

These results can be explained by looking in more detail at the kinetic parameters of NamPRT and NAD-consuming enzymes such as Sirtuin~1. Most NAD-consuming enzymes are inhibited by their product Nam. Thus the presence of NNMT enables higher NAD consumption fluxes, by removing excess Nam from the cells. At the same time, the high substrate affinity of NamPRT maintains a sufficiently high NAD concentration, although the concentration is, as expected, lower than in the system without NNMT.

As measured kinetic parameters of NamPRT are only available for the human enzyme \cite{Burgos2008}, we analysed the potential effect of NamPRT affinity ($K_{M}$) on NAD steady state concentration and NAD consumption flux. In the absence of NNMT, a variation of the substrate affinity of NamPRT for Nam, has very little effect on steady state NAD concentration and NAD consumption flux (fig.~\ref{fig:NamPRT_affinity_Nam}A and~B). In the presence of NNMT, however, NAD consumption flux and NAD concentration increases with decreasing $K_{M}$ values (fig.~\ref{fig:NamPRT_affinity_Nam}C and~D).

Remarkably, NAD concentration and consumption flux are both considerably affected by cell division rates in a system without NNMT, at least if the enzyme expression is kept constant at different cell division rates. Of course, this is an artificial scenario, as one would assume organisms to regulate enzyme expression to achieve similar levels of metabolite concentrations instead. Nevertheless, in the absence of NNMT there seems to be a tradeoff between maintainable NAD concentration and consumption flux. In contrast, in the presence of NNMT, NAD consumption rates and concentrations are almost independent of cell division rates.

Figures~\ref{fig:NamPRT_affinity_Nam}E and~F show a direct comparison of different affinities of NamPRT for Nam with and without NNMT. At an affinity of $K_{M}$ =~1\,$\mu$M, which is in the range of the $K_{M}$ of NADA for Nam, NAD consumption flux is only higher with NNMT than without at low cell division rates (fig.~\ref{fig:NamPRT_affinity_Nam}E). If the affinity of NamPRT is high enough, consumption rates are always higher with NNMT than without it. The NAD concentration is always lower with NNMT (fig.~\ref{fig:NamPRT_affinity_Nam}F).

As pathway dynamics are not solely dependent on one enzyme, we wanted to estimate the impact of the substrate affinity of NNMT that is competing with NamPRT for the same substrate. The substrate affinity values found in the human enzymes (indicated by black asterisks in fig.~\ref{fig:optimal_substr_affinities}) are actually optimal with respect to both achievable steady state NAD concentration and consumption fluxes. Thus, a further increase of the affinity of NamPRT for Nam or a change in the affinity of NNMT for Nam would not provide any further advantage.


\subsection{Sequence variance acquired in metazoans enhances substrate affinity}

As our simulations suggested that NNMT might have exerted an evolutionary pressure on the development of NamPRT, we created a multiple sequence alignment to see if we can find sequence variations in NamPRT upon the appearance of NNMT. The multiple sequence alignment of selected sequences is shown in figure~\ref{fig:unresolved_loop}A and a more comprehensive multiple sequence alignment containing a larger number of species can be found in supplementary figure~S1. We found that most Deuterostomes that possess only NamPRT and NNMT (indicated by the blue circle) have an insert of ten amino acids corresponding to positions 42 to 51 in the human enzyme. This insert overlaps with a predicted weak nuclear localisation signal (NLS), that is lost when the insert is removed. The sequence corresponds to a loop at the protein surface that is unresolved in all available crystal structures of human NamPRT (e.g. structure visualisation in fig.~\ref{fig:unresolved_loop}B from \cite{Wang2006}). The loop is connected to one of the $\beta$-sheets involved in substrate binding \cite{Burgos2009}.

From these observations, we derived two possible hypotheses regarding the role of the loop in NamPRT function. The first hypothesis was that the presence of the loop changes the subcellular localisation of NamPRT, as it is overlapping with a predicted NLS. To test this hypothesis, we created a mutant NamPRT lacking the loop and recombinatly expressed FLAG-tagged wildtype and mutant NamPRT in Hela S3 cells. Immunofluorescence imaging showed a mixed cytosolic nuclear localisation for the wildtype and the mutant NamPRT (fig.~\ref{fig:unresolved_loop}C). Thus, the presence of the loop does not compromise the nuclear localisation.

The second hypothesis was that the sequence insertion influences the affinity of NamPRT, which would be in line with the predictions from our simulations. We recombinantly expressed both wildtype and mutant NamPRT in \textit{E.~coli}. After purification (suppl. fig.~S3), we measured the activity using NMR detection of NMN in the presence and absence of ATP. The enzymatic activity of the mutant enzyme lacking the loop was much lower than the activity of the wildtype enzyme (fig.~\ref{fig:unresolved_loop}D). Furthermore, an increase of the substrate concentration had a much stronger effect on the activity of the mutant compared to the wildtype enzyme, indicating a lower affinity. In contrast to wildtype NamPRT, the mutant did not exhibit an increased activity in the presence of ATP (fig.~\ref{fig:unresolved_loop}E).


\subsection{NamPRT and NNMT make NADA obsolete in vertebrates}

A remaining puzzle was, why NADA is lost in vertebrates, while NADA and NamPRT both occur in bacteria. We tried to approach the puzzle with a competitive model with two compartments that share a Nam source. One compartment contains NADA while the other contains either NamPRT alone, or together with NNMT (for details, see experimental procedures and suppl. tab.~S2). Without NNMT, the compartment containing NADA shows slightly lower NAD consumption rates (fig.~\ref{fig:NNMT_comp_advantage}A), but is able to maintain much higher NAD concentrations at low cell division rates (fig.~\ref{fig:NNMT_comp_advantage}B). At high cell division rates, steady state concentrations in both compartments are similar. These comparable concentrations might explain why in bacteria that often have relatively high growth rates, both systems co-exist.

In the presence of NNMT, the NamPRT compartment has both higher NAD consumption rates and higher steady state NAD concentrations than the compartment containing NADA (fig.~\ref{fig:NNMT_comp_advantage}C and~D). The high NAD concentrations are dependent on a high affinity of NamPRT. When simulating a low affinity (high $K_{M}$) of NamPRT, the NADA compartment is able to maintain higher NAD concentrations although it still has a lower the NAD consumption flux. Taken together, the results suggest that the NADA pathway became less worthwhile when a higher affinity of NamPRT developed, possibly induced by NNMT.
