\section{Results}

\subsection{Phylogenetic analysis of NAD biosynthesis and consumption}

As shown in Fig. 1, NAD can be synthesized using several routes from three main precursors: tryptophan, nicotinamide (Nam) and nicotinic acid (NA). Nam and NA are together known as vitamin B3 or niacin. Alternatively, nicotinamide ribose (NR) can be used omitting the energetically unfavourable reaction of Nam phosphoribosyltransferase (NamPRT), requiring nicotinamide ribose kinase (NRK) instead \cite{Imai2018}. Du to the hight turnover of NAD observed and the fact that only 1\% of the tryptophan taken up with our diet is converted into NAD, vitamin B3 (Nam and NA) and to a lower extend NR are the major precursor of NAD-biosynthesis. 

For the recycling of Nam, two different pathways exist. The pathway found in yeast and plants is using a four-step pathway starting with the deamination of Nam to nicotinic acid by NADA. The other three enzymes comprise the Preiss-Handler pathway that also exists in vertebrates. The recycling pathway found in mammals directly converts Nam into the corresponding mononucleotide (NMN) a reaction catalysed by NamPRT and driven by a non-stoichiometric ATP-hydrolysis. A similar reaction catalysed by an evolutionary related enzyme NAPRT, converts NA into the NA mononucleotide in the Preiss-Handler pathway. NMN and NAMN are converted into dinucleotides by the Nam/NA adenylytransferases (NMNATs). The recycling pathway via NA finally requires an amination step catalysed by NADsynthase, driven by the conversion of ATP to AMP producing pyrophosphate. Even though the latter pathway seems to be very inefficient, it is the pathway preferentially used by most bacteria, fungi and plants (see Figure 2A), whereas most metazoans recycle Nam using NamPRT.

Analysing the phylogeny of the NAD recycling enzymes in Metazoa in more detail reveals that not only does NamPRT replace NADA, but in most organisms, especially in Deuterostomia, NamPRT is found together with the Nam methyltransferase NNMT (Figure 2B). NNMT methylates Nam to methyl-Nam that is in mammals excreted with the urine, thus removing Nam from recycling. NNMT seems to have arisen de novo in the common ancestor of Ecdysozoa and Lophotrochozo.  We were not be able to find any gene with considerable similarity in fungi or plants, even though in fungi a gene named NNMT can be found in databases. These genes do, however, show very limited sequence similarities to the NNMT of nematodes or deuterostomes and in the yeast protein has later on be found to be a lysin-protein-methyltransferase  \cite{?}(\todo[author=Mathias B.]{Please look up the reference for this.}. Nematodes are the only organisms where we find NNMT together with NADA without NamPRT being present. In Deuterostomia the only large class that does only have NamPRT and seems to have lost NNMT again are Sauropsida and here especially birds. The reason why a lot of birds do not encode NNMT remains unclear, as the appearance is quite scattered (not shown)\todo[author=Mathias]{Maybe as supplementary figure?}. It might be related to the excretion system, as the product of NNMT methyl-nicotinamide is in mammals excreted with the urine. There are some species where we could not find NamPRT or NADA but NNMT, we assume that this is due to incomplete genomes in the database.

In addition to the phylogenetic distribution of the two Nam salvage enzymes NADA and NamPRT, we looked at phylogenetic diversity of enzymes catalysing NAD-dependent signalling reactions. To do so we used the previously established classification into 10 different families \cite{Gossmann2012} (For details see materials and methods and supplementary information). The numbers in Figure 2B denote the average number of NAD-dependent signalling enzyme families found in each taxa. With the exception of Cnidaria and Lophotrochozoa, we find average 3 to 4 families in Protostomia, whereas most Deuterostomia have on average more than 8 families with an increasing diversification of enzymes within some of these families \cite{Gossmann2014}.

Taken together, we found that the presence of NamPRT and NNMT coincides with an increased diversification of NAD-dependent signalling, whilst NADA is lost in vertebrates. That is  counterintuitive, as one would expect that a decrease in precursor concentration caused by the precursor removal through NNMT, should cause a decrease of NAD availability and consequently less active NAD dependent signalling.


\subsection{Dynamics of NAD biosynthesis and consumption}

So why does the diversity of NAD-dependent signalling increase? And why does NADA disappear in Deuterostomia although it is the predominant pathway in bacteria, plants and fungi? Given the complexity of the NAD metabolic network, this question is difficult to be comprehensively addressed experimentally. Thus, to answer these questions we built a dynamic model of NAD metabolism using existing kinetic data from the literature (details see materials and methods and supplementary material).

To be able to compare metabolic features of evolutionary quite different systems in our simulations, and as we have limited information about expression levels of enzymes or changes of kinetic constants during evolution, we initially used the kinetic constants found for yeast or human enzymes for all systems analysed and used equal amounts of enzymes for all reactions. Wherever possible we did not only include substrate affinities but also known product inhibition or inhibition by downstream metabolites. As we assume that cell growth is, besides NAD-consuming reactions, a major driving force for NAD biosynthesis, we analysed different growth rates (cell division rates) by simulating different dilution rates for all metabolites. We assume furthermore that Nam availability is different for different organisms and thus in addition analysed this simulating different Nam import rates.

The pathway using NADA and recycling Nam via NA is superior \todo[author=Mathias]{What is meant with superior? NAD consumption flux is higher or lower with NNMT, depending on the condition. Free NAD concentration is higher without NNMT, but is that superior?} both in terms of steady state NAD concentration and NAD consumption flux (representing the activity of NAD-dependent signalling) in the absence of NNMT (Figure 3). In the presence of NNMT, the picture is slightly different (Figure 3C-D), even though NADA is still superior to NamPRT under most conditions \todo[author=Ines]{Check if not under all shown}\todo[author=Mathias]{Reply: From fig 4C, it looks like the flux is decreased at very high cell division rates while the Nam import rate is low.}. Only if Nam availability is very low, NamPRT is performing better because of its high substrate affinity. However, the disadvantage of NamPRT at equal amounts of enzyme, can be compensated by an increased expression of NamPRT. Although, to reach similar NAD concentrations in our model, NamPRT expression has to be tenfold higher than the expression of NAPRT (Figure S1). The NADA expression required is very low due to its high turnover. This might provide an explanation why we find NADA and with that the pathway via NA predominantly in bacteria, yeast, and plants, organisms that show high cell division rates. Under these conditions, protein expression costs are assumed to have a high impact on metabolic performance \todo[author=Mathias]{source} and thus pathways where low enzyme expression suffices, might be favourable, even though the pathway via NA is energetically less efficient.

NADA seems to be able to maintain higher NAD concentrations than NamPRT. This figure changes if we simulate two organisms or cells that are in direct competition for Nam. Under these conditions, the organism expressing NamPRT has a competitive advantage above NADA-expressing organisms, but this only holds if the competing organisms expresses NamPRT together with NNMT. (new Figure) This observation might explain, why NADA disappeared in Metazoa together with the appearance of NNMT. It might also explain, why many bacteria associated with mammals harbour NamPRT instead of NADA\todo[author=Mathias]{Needs some figure or source}.

Although these simulations already provide an idea of why we find NamPRT predominantly in combination with NNMT, we still do not know why this development coincides with an increased diversification of NAD-consuming enzymes. When we simulate the presence or absence of NNMT in the presence of NamPRT, we see that the impact of NNMT on NAD concentration is relatively small (Figure 4D), but we see that that NNMT increases the NAD consumption flux under most conditions in the presence of NamPRT (Figure 4C). NAD consumption flux can be further increased by increasing the expression NamPRT, which also compensates the slight reduction in NAD concentration in the presence of NNMT (see Figure S2).

These findings can be explained when looking in more detail into the kinetic parameters of NamPRT and NAD-consuming enzymes such as Sirt1p. The ability to maintain high a NAD concentration in the presence of NNMT and at low Nam availability, is due to the very high affinity of NamPRT for its substrate, having a half saturation constant (Km) in the low nanomolar range. The increase of NAD consumption flux is caused by the fact that most NAD-consuming enzymes are inhibited by their product Nam, which is also the reason why the presence of NNMT enables higher NAD consumption fluxes.

As the substrate affinity of NamPRT for Nam is extremely high (with a Km in the low nanomolar range) and as this might not have been the case throughout evolution, we analysed the effect of the NamPRT Km on NAD steady state concentration and NAD consumption flux, leaving all other kinetic parameters constant. In the absence of NNMT (Figure 5A-B) the Km has very little effect on steady state NAD concentration and NAD consumption flux. Without NNMT, NAD concentration and consumption flux are both considerably affected by cell division rates, at least if the enzyme expression is kept constant. This is of course an artificial scenario, as one would assume organisms to regulate enzyme expression to achieve similar levels of metabolite concentrations instead. In the absence of NNMT there appears furthermore to be a trade-off between achievable steady state NAD concentration and NAD consumption flux.

In the presence of NNMT, NAD consumption flux and NAD concentration increases with decreasing Km values (Figure 5C-D). And we note, that both the NAD steady state concentration and the consumption flux are relatively stable over a wide range of cell division rates even at constant expression levels of the involved enzymes, suggesting that NNMT might have an important role to maintain homeostasis of NAD metabolism.

When we compare NAD consumption and NAD concentration with and without NNMT with two different substrate affinities of NamPRT, we see that at a low affinity (Km of 100 nM, which is in the range of the Km of NAPRT for its substrate, or the Km of NADA for Nam), NAD consumption flux is only higher with NNMT at low cell division rates, whereas at high division rates, higher NAD consumption fluxes are achieved without NNMT (Figure 5 E-F). This might explain why we do not find NNMT in organism that tend to have high growth rates\todo[author=Ines]{I am not sure if we should go even further suggesting that decreasing substrate affinity by using a competitive inhibitor such as FK866 will influence the consumption rate in fast growing cells much more than the one in slow growing cells, being potentially relevant for cancer therapy. – Discussion}. We furthermore find, that the competitive advantage of NamPRT of NADA is only present at sufficiently high affinity of NamPRT.

Our analysis also suggests that NNMT might have exerted an evolutionary driving force on the substrate affinity of NamPRT, explaining the extreme values found for the human enzyme\todo[author=Ines]{Move to discussion?}.

The pathway dynamics are of course not solely dependent on one enzyme. Thus, what is the impact of the substrate affinity of NNMT that is competing with NamPRT for the same substrate? In Figure 6A we see that the substrate affinity values found in the human enzymes (indicated by black asterisks) are actually optimal with respect to both achievable steady state NAD concentration and consumption fluxes. Thus, a further increase of the affinity of NamPRT for Nam would not provide any advantage.


\subsection{Sequence variance acquired in metazoans enhances substrate affinity}

To see whether we can find sequence variations in the protein sequence of NamPRT that indicate evolutionary changes of NamPRT upon the appearance of NNMT, we created a multiple sequence alignment of selected eukaryotic NamPRT sequences. As shown in Figure 7A and Supplementary Figure 3\todo[author=Ines]{To be created}, Deuterostomia that have only NamPRT and NNMT (indicated by the number 6 in parenthesis \todo[author=Ines]{Maybe rather indicate this by colour and the kingdom through abbrev?}) have an insert of ten amino acids corresponding to positions 43 to 52 of the human enzyme. Looking at the crystal structure of human NamPRT this sequence insertion corresponds to a region that has not been structurally resolved in any of the currently available crystal structures (e.g. \cite{Wang2006} structure visualisation Figure 7B) and its function is unclear. The unresolved loop structure overlaps with a predicted weak nuclear localisation signal (figure 7A), that is not present without the insertion. The loop is in addition connected to one of the beta-sheets involved in substrate binding, potentially affecting the affinity or turnover of the enzyme.

The observed evolutionary change in the primary sequence of NamPRT could therefore have had different effects, that we wanted to test experimentally. We first investigated whether the deletion of the amino acids 43 to 52 has an effect on the localisation of the human enzyme, expressing a …. mutant lacking this sequence in HeLa cells. We could, however, not detect any changes in subcellular localisation (Figure 7C) compared to the wildtype fusion construct. We therefore conclude that the partial nuclear localisation of NamPRT is not affected by the deletion. We then tested the enzymatic activity of the NamPRT mutant recombinantly expressed in E. coli. As shown in Figure 7D the enzyme still forms a dimer, thus seems to be folded correctly. The enzymatic activity is, however, much lower compared to the wildtype enzyme. Using higher concentrations of Nam, it appears that the mutant is not saturated at 100 \textmu M, pointing to a decreased substrate affinity of the mutant enzyme.