%%% TeX-master: "manuscript"

\section{Results}

\subsection{Phylogenetic analysis of NAD biosynthesis and consumption}

As shown in Fig. 1, NAD can be synthesized using several routes from three main precursors: tryptophan, nicotinamide (Nam) and nicotinic acid (NA). Nam and NA are together known as vitamin B3 or niacin. Alternatively, nicotinamide ribose (NR) can be used omitting the energetically unfavourable reaction of Nam phosphoribosyltransferase (NamPRT), requiring nicotinamide ribose kinase (NRK) instead \cite{Yoshino2018}. Du to the hight turnover of NAD observed and the fact that only 1\% of the tryptophan taken up with our diet is converted into NAD, vitamin B3 (Nam and NA) and to a lower extend NR are the major precursor of NAD-biosynthesis.

For the recycling of Nam, two different pathways exist. The pathway found in yeast and plants is using a four-step pathway starting with the deamination of Nam to nicotinic acid by NADA. The other three enzymes comprise the Preiss-Handler pathway that also exists in vertebrates. The recycling pathway found in mammals directly converts Nam into the corresponding mononucleotide (NMN) a reaction catalysed by NamPRT and driven by a non-stoichiometric ATP-hydrolysis. A similar reaction catalysed by an evolutionary related enzyme NAPRT, converts NA into the NA mononucleotide in the Preiss-Handler pathway. NMN and NAMN are converted into dinucleotides by the Nam/NA adenylytransferases (NMNATs). The recycling pathway via NA finally requires an amination step catalysed by NADsynthase, driven by the conversion of ATP to AMP producing pyrophosphate. Even though the latter pathway seems to be very inefficient, it is the pathway preferentially used by most bacteria, fungi and plants (see Figure 2A), whereas most metazoans recycle Nam using NamPRT.

Analysing the phylogeny of the NAD recycling enzymes in Metazoa in more detail reveals that not only does NamPRT replace NADA, but in most organisms, especially in Deuterostomia, NamPRT is found together with the Nam methyltransferase NNMT (Figure 2B). NNMT methylates Nam to methyl-Nam that is in mammals excreted with the urine, thus removing Nam from recycling. NNMT seems to have arisen de novo in the common ancestor of Ecdysozoa and Lophotrochozo.  We were not be able to find any gene with considerable similarity in fungi or plants, even though in fungi a gene named NNMT can be found in databases. These genes do, however, show very limited sequence similarities to the NNMT of nematodes or deuterostomes and in the yeast protein has later on be found to be a lysin-protein-methyltransferase \cite{Wlodarski2011}. Nematodes are the only organisms where we find NNMT together with NADA without NamPRT being present. In Deuterostomia the only large class that does only have NamPRT and seems to have lost NNMT again are Sauropsida and here especially birds. The reason why a lot of birds do not encode NNMT remains unclear, as the appearance is quite scattered (not shown)\todo[author=Mathias]{Maybe as supplementary figure?}. It might be related to the excretion system, as the product of NNMT methyl-nicotinamide is in mammals excreted with the urine. There are some species where we could not find NamPRT or NADA but NNMT, we assume that this is due to incomplete genomes in the database.

In addition to the phylogenetic distribution of the two Nam salvage enzymes NADA and NamPRT, we looked at phylogenetic diversity of enzymes catalysing NAD-dependent signalling reactions. To do so we used the previously established classification into 10 different families \cite{Gossmann2012FEBS} (For details see materials and methods and supplementary information). The numbers in Figure 2B denote the average number of NAD-dependent signalling enzyme families found in each taxa. With the exception of Cnidaria and Lophotrochozoa, we find average 3 to 4 families in Protostomia, whereas most Deuterostomia have on average more than 8 families with an increasing diversification of enzymes within some of these families \cite{Gossmann2014DNAR}.

Taken together, we found that the presence of NamPRT and NNMT coincides with an increased diversification of NAD-dependent signalling, whilst NADA is lost in vertebrates. That is  counterintuitive, as one would expect that a decrease in precursor concentration caused by the precursor removal through NNMT, should cause a decrease of NAD availability and consequently less active NAD dependent signalling.


\subsection{Dynamics of NAD biosynthesis and consumption}

So why does the diversity of NAD-dependent signalling increase? And why does NADA disappear in Deuterostomia although it is the predominant pathway in bacteria, plants and fungi? Given the complexity of the NAD metabolic network, this question is difficult to be comprehensively addressed experimentally. Thus, to answer these questions we built a dynamic model of NAD metabolism using existing kinetic data from the literature (for details see materials and methods and supplementary material).

To be able to compare metabolic features of evolutionary quite different systems in our simulations, and as we have limited information about expression levels of enzymes or changes of kinetic constants during evolution, we initially used the kinetic constants found for yeast or human enzymes for all systems analysed and used equal amounts of enzymes for all reactions. Wherever possible we did not only include substrate affinities but also known product inhibition or inhibition by downstream metabolites. As we assume that cell growth is, besides NAD-consuming reactions, a major driving force for NAD biosynthesis, we analysed different growth rates (cell division rates) by simulating different dilution rates for all metabolites.

First we addressed the question of the predominant coexitence of NamPRT and NNMT in vertebrates that coincides with an increase in the number of NAD-consuming enzyme families. We thus used our dynamic model of the NAD biosynthesis and consmption pathway to simulate the effect of the  presence or absence of NNMT.  We see that the presence of NNMT enables higher NAD consumption rates  (Figure3A) while, as exptected, reducing the steady state concentration of NAD (Figure 3B). The decline in NAD-concnetrtion can be compensated by a higher expression of NamPRT (not shown) further increasing NAD consumption flux.

These findings can be explained when looking in more detail into the kinetic parameters of NamPRT and NAD-consuming enzymes such as Sirt1p.  The increase of NAD consumption flux is caused by the fact that most NAD-consuming enzymes are inhibited by their product Nam, explaing why the presence of NNMT enables higher NAD consumption fluxes. At the same time, the high substrate affinity of NamPRT maintains a sufficiently high NAD-concetration.

As kinetic parameters of NamPRT are only available for the human enzyme \cite{Burgos2008} , we went on to theoretically analysed the potential effect of the NamPRT affinity (Km) on NAD steady state concentration and NAD consumption flux, leaving all other kinetic parameters constant. In the absence of NNMT (Figure 3C-D) a change in Km has very little effect on steady state NAD concentration and NAD consumption flux.  In the presence of NNMT, however, NAD consumption flux and NAD concentration increases with decreasing Km values (Figure 5E-F).

It is intersting to note that, without NNMT, NAD concentration and consumption flux are both considerably affected by cell division rates, at least if the enzyme expression is kept constant. Even though, this is of course an artificial scenario, as one would assume organisms to regulate enzyme expression to achieve similar levels of metabolite concentrations instead. However, there seems to be an apparent trade off between maintainable NAD concentration and consumption flux.

When we compare NAD consumption and NAD concentration with and without NNMT with two different substrate affinities of NamPRT, we see that at a low affinity (Km of 1 mikroM, which is in the range of the Km of NADA for Nam, NAD consumption flux is only higher with NNMT at low cell division rates. This effect becomes stronger if we further lower the affinity (not shown) (Figure 3G-H).


The pathway dynamics are of course not solely dependent on one enzyme. Thus, what is the impact of the substrate affinity of NNMT that is competing with NamPRT for the same substrate? In Figure 4A we see that the substrate affinity values found in the human enzymes (indicated by black asterisks) are actually optimal with respect to both achievable steady state NAD concentration and consumption fluxes. Thus, a further increase of the affinity of NamPRT for Nam would not provide any advantage.


\subsection{Sequence variance acquired in metazoans enhances substrate affinity}

As our simulations suggest that NNMT might have excerted an evolitionary pressure on the development of NamPRT. We performed multiple sequence analysis to see if we can find sequence variations in the protein sequence of NamPRT that indicate evolutionary changes of NamPRT upon the appearance of NNMT. The multiple sequence alignment of selected sequences is shown in Figure 5A and Supplementary Figure ?\todo[author=Ines]{Decide figure number} We recognized that Deuterostomia that have only NamPRT and NNMT (indicated by the blue circle) have an insert of ten amino acids corresponding to positions 43 to 52 of the human enzyme, that overlaps with a predicted weak nuclear localisation signal, that is lost when removing this sequence stretch.
Furthermore, the inserted sequence corresponce to an structurallly unresolved loop structure in all avaibable crystal structure of human NamPR (e.g. \cite{Wang2006} structure visualisation Figure 5B). The loop is connected to one of the beta-sheets involved in substrate binding.

From these observations we derived two hypotheses regarding the potential function of the loop for NamPRT function. Firstly, we analysed whether the deletion of the amino acids 43 to 52 has an effect on the localisation of the human enzyme. But both wildtype and mutant protein show a mixed localisation in the cytosol and nucleus (Figure 5C). We therefore conclude that the partial nuclear localisation of NamPRT is not affected by the sequence deletion.
The second hypothesis was that the sequence insertion influences NamPRT affinity and activity, which is also what our mathematical simulations would predict. We thus recombinantly expressed  both wildttype and mutant  NamPRT  in E. coli.  Upon purifictaion we meassured the activity using NMR detecttion of NMN produced in the presence and absence of ATP. It appears that the enzymatic activity of the mutant enzyme is much lower than that of the wildtype enzyme both with and without ATP. We furthermore see that the activity of the mutant enzyme can be increase with higher concentration of Nam and PRPP whereas that is not the case for the wildtype enzyme. Pointing to a decrease substrate affinity of the mutant enzyme. Supporting the second hypothesis and thus supporting the predictions derived from our mathematical modelling approach.

\subsection{Loss of NADA in vertebrates}

But why is NADA lost in vertebrates? Our simulations show that in direct competition NADA is outcompeted by the combined presence of NamPRT and NNMT but not by NamPRT alone. This effect deisappears of NamPRT affinity is lowered. ....

