%%% TeX-master: "manuscript"
% !TeX spellcheck=en_GB

\section{Results}

\subsection{Phylogenetic analysis of NAD biosynthesis and consumption}

NAD can be synthesised using several routes from three main precursors: tryptophan, nicotinamide (Nam) and nicotinic acid (NA) (Overview see fig.~\ref{fig:pathway_overview}). Nam and NA are together known as vitamin B3 or niacin. Alternatively, nicotinamide ribose (NR) can be used omitting the energetically unfavourable reaction of Nam phosphoribosyltransferase (NamPRT), requiring nicotinamide ribose kinase (NRK) instead \cite{Yoshino2018}. Due to the high turnover of NAD observed and the fact that only 1\% of the tryptophan taken up with our diet is converted into NAD\todo{source}, vitamin B3 (Nam and NA) and to a lower extend NR are the major precursors of human NAD biosynthesis.

Looking across all known species, two different pathways exist that recycle Nam. The major pathway found in yeast and plants uses a four-step pathway starting with the deamination of Nam to nicotinic acid by NADA. The other three enzymes comprise the Preiss-Handler pathway that also exists in vertebrates. The recycling pathway found in mammals directly converts Nam into the corresponding mononucleotide (NMN) a reaction catalysed by NamPRT and driven by a non-stoichiometric ATP-hydrolysis. A similar reaction catalysed by an evolutionary related enzyme NAPRT, converts NA into the NA mononucleotide in the Preiss-Handler pathway. NMN and NAMN are converted into dinucleotides by the Nam/NA adenylytransferases (NMNATs). The recycling pathway via NA finally requires an amidation step catalysed by NAD synthase, driven by the conversion of ATP to AMP producing pyrophosphate. Even though the latter pathway seems to be very inefficient, it is the pathway preferentially used by most bacteria, fungi and plants (see fig.~\ref{fig:phylo_distribution}A), whereas most metazoans recycle Nam using NamPRT.

Analysing the phylogeny of the NAD recycling enzymes in Metazoa in more detail reveals that not only NamPRT replaces NADA, but in most organisms, especially in Deuterostomia, NamPRT is found together with the Nam methyltransferase NNMT (fig.~\ref{fig:phylo_distribution}B). NNMT methylates Nam to methyl-Nam that is in mammals excreted with the urine, thus removing Nam from recycling. NNMT seems to have arisen \textit{de novo} in the common ancestor of Ecdysozoa and Lophotrochozoa. We were not be able to find any gene with considerable similarity in fungi or plants, even though in fungi genes named NNMT can be found in databases. These genes do, however, show very limited sequence similarities to the NNMT of nematodes or deuterostomes. The yeast protein has later been shown to be a lysin-protein-methyltransferase \cite{Wlodarski2011}. Nematodes are the only organisms that encode NNMT together with NADA without NamPRT being present. In Deuterostomia the only large class that does only have NamPRT and seems to have lost NNMT again are Sauropsida and here especially birds. The reason why a lot of birds do not encode NNMT remains unclear, as the appearance is quite scattered (figure S2\todo[author=Mathias B.]{Has to be made!}). The lack of NNMT might be related to the excretion system, as the product of NNMT methyl-Nam is in mammals excreted with the urine. There are some species where we could not find NamPRT or NADA but NNMT, we assume that this is due to incomplete genomes in the database.

In addition to the phylogenetic distribution of the two Nam salvage enzymes NADA and NamPRT, we looked at phylogenetic diversity of enzymes catalysing NAD-dependent signalling reactions. To do so we used the previously established classification into ten different enzyme families \cite{Gossmann2012FEBS}. The detailed list of templates used for the phylogenetic analysis can be found in supplementary table~S1. The numbers in figure~\ref{fig:phylo_distribution}B denote the average number of NAD-dependent signalling enzyme families found in each taxa. With the exception of Cnidaria and Lophotrochozoa, we find average three to four families in Protostomia, whereas most Deuterostomia have on average more than eight families with an increasing diversification of enzymes within some of these families \cite{Gossmann2014DNAR}.

Taken together, we found that the presence of NamPRT and NNMT coincides with an increased diversification of NAD-dependent signalling, whilst NADA is lost in vertebrates. This seems counterintuitive, as one would expect that a decrease in precursor concentration caused by the precursor removal through NNMT, should cause a decrease of NAD availability and consequently less active NAD dependent signalling.


\subsection{Dynamics of NAD biosynthesis and consumption}

So why does the diversity of NAD-dependent signalling increase? And why does NADA disappear in Deuterostomia although it is the predominant pathway in bacteria, plants and fungi? Given the complexity of the NAD metabolic network, these questions are difficult to be comprehensively addressed experimentally. We therefore built a dynamic model of NAD metabolism using existing kinetic data from the literature (for details see experimental procedures and supplementary table~S2).

To be able to compare metabolic features of evolutionary quite different systems in our simulations, and as we have limited information about expression levels of enzymes or changes of kinetic constants during evolution, we initially used the kinetic constants found for yeast or human enzymes for all systems analysed and used equal amounts of enzymes for all reactions. Wherever possible, we did not only include substrate affinities but also known product inhibitions or inhibition by downstream metabolites. As we assume that cell growth is, besides NAD-consuming reactions, a major driving force for NAD biosynthesis, we analysed different growth rates (cell division rates) by simulating different dilution rates for all metabolites.

First, we addressed the question of the predominant co-existence of NamPRT and NNMT in vertebrates that coincides with an increase in the number of NAD-consuming enzyme families. We thus used our dynamic model to simulate steady state concentrations and NAD consumption rates simulating NAD biosynthesis via NamPRT in the presence or absence of NNMT. We see that the presence of NNMT enables higher NAD consumption rates (fig.~\ref{fig:NNMT_NAD_flux}A) while, as expected, reducing the steady state concentration of NAD (fig.~\ref{fig:NNMT_NAD_flux}B). The decline in NAD concentration can be compensated by a higher expression of NamPRT, further increasing NAD consumption flux (dashed lines in fig.~\ref{fig:NNMT_NAD_flux}A and~B).

These findings can be explained when looking in more detail at the kinetic parameters of NamPRT and NAD-consuming enzymes such as Sirtuin~1. The increase of NAD consumption flux is caused by the fact that most NAD-consuming enzymes are inhibited by their product Nam, explaining why the presence of NNMT enables higher NAD consumption fluxes. At the same time, the high substrate affinity of NamPRT maintains a sufficiently high NAD concentration.

As kinetic parameters of NamPRT are only available for the human enzyme \cite{Burgos2008}, we went on to theoretically analyse the potential effect of NamPRT affinity ($K_{M}$) on NAD steady state concentration and NAD consumption flux. In the absence of NNMT (fig.~\ref{fig:NamPRT_affinity_Nam}A and~B) a change in $K_{M}$ has very little effect on steady state NAD concentration and NAD consumption flux. In the presence of NNMT, however, NAD consumption flux and NAD concentration increases with decreasing $K_{M}$ values (fig.~\ref{fig:NamPRT_affinity_Nam}C and~D).

It is interesting to note that without NNMT, NAD concentration and consumption flux are both considerably affected by cell division rates, at least if the enzyme expression is kept constant. This is, of course, an artificial scenario, as one would assume organisms to regulate enzyme expression to achieve similar levels of metabolite concentrations instead. However, there seems to be an apparent trade off between maintainable NAD concentration and consumption flux in the absence of NNMT. In contrast,  in the presence of NNMT NAD consumption rates increase with NAD concentration and both are relatively independent of cell division rates. \todo{The last sentence is unclear to me. Consumption increases with concentration? That is not in the figure. I needed to read often to understand what "NNMT and both" means.}

When we compare NAD consumption and NAD concentration with and without NNMT with two different substrate affinities of NamPRT, we see that at low affinity ($K_{M}$ of 1 $\mu$M, which is in the range of the $K_{M}$ of NADA for Nam), NAD consumption flux is only higher with NNMT at low cell division rates. This effect becomes stronger with decreasing affinity (fig.~\ref{fig:NamPRT_affinity_Nam}E and~F).

The pathway dynamics are not solely dependent on one enzyme. Thus, what is the impact of the substrate affinity of NNMT that is competing with NamPRT for the same substrate? In figure~\ref{fig:optimal_substr_affinities} we see that the substrate affinity values found in the human enzymes (indicated by black asterisks) are actually optimal with respect to both achievable steady state NAD concentration and consumption fluxes. Thus, a further increase of the affinity of NamPRT for Nam would not provide any advantage.


\subsection{Sequence variance acquired in metazoans enhances substrate affinity}

As our simulations suggest that NNMT might have exerted an evolutionary pressure on the development of NamPRT, we created a multiple sequence alignment to see if we can find sequence variations in the protein sequence of NamPRT that indicate evolutionary changes of NamPRT upon the appearance of NNMT. The multiple sequence alignment of selected sequences is shown in figure~\ref{fig:unresolved_loop}A and a more comprehensive multiple sequence alignment containing a larger number of species can be found in supplementary figure~S1. We recognised that most Deuterostomia that have only NamPRT and NNMT (indicated by the blue circle) have an insert of ten amino acids corresponding to positions 42 to 51 of the human enzyme. This insert overlaps with a predicted weak nuclear localisation signal, that is lost when removing this sequence stretch. The inserted sequence corresponds to a structurally unresolved loop in all available crystal structure of human NamPRT (e.g. \cite{Wang2006} structure visualisation fig.~\ref{fig:unresolved_loop}B). The loop is connected to one of the $\beta$-sheets involved in substrate binding.

From these observations, we derived two possible hypotheses regarding the role of the loop for NamPRT function. Firstly, we analysed whether the deletion of the amino acids 42 to 51 has an effect on the localisation of the human enzyme. We thus performed immunofluorescence imaging with FLAG-tagged proteins, but both wildtype and mutant protein showed a mixed cytosolic nuclear localisation (fig.~\ref{fig:unresolved_loop}C). We therefore conclude that the partial nuclear localisation of NamPRT is not affected by the sequence deletion.

The second hypothesis was that the sequence insertion influences NamPRT affinity, which is what our mathematical simulations would predict. We thus recombinantly expressed both wildtype and mutant NamPRT in \textit{E.~coli}. Upon purification, we measured the activity using NMR detection of NMN produced in the presence and absence of ATP. It appears that the enzymatic activity of the mutant enzyme is much lower than that of the wildtype enzyme (fig.~\ref{fig:unresolved_loop}D) and that an increase of the substrate concentration has a much stronger effect on the activity of the mutant compared to the wildtype enzyme, indicating a lower affinity. We furthermore showed that in contrast to NamPRT wildtype, the mutant does not exhibit an increased activity in the presence of ATP (fig.~\ref{fig:unresolved_loop}E).


\subsection{Loss of NADA in vertebrates}

The remaining question is now, why NADA is lost in vertebrates, while NADA and NamPRT are co-existing in bacteria. To analyse this question we built a two compartment model that has a shared Nam source (details see experimental procedures and supplementary table~S2). One compartment contains NADA while the other contains either NamPRT alone at equal expression levels, or together with NNMT. Without NNMT the compartment containing NADA has slightly lower NAD consumption rates (fig.~\ref{fig:NNMT_comp_advantage}A), but is able to maintain much higher steady state NAD concentrations at low cell division rates (fig.~\ref{fig:NNMT_comp_advantage}B). At high cell division rates, steady state concentrations in both compartments are similar. The latter might explain why in bacteria that have relatively high growth rates both systems co-exist.

In the presence of NNMT, the NamPRT compartment out-competes the NADA-containing compartment, both with respect to NAD consumption rates and steady state NAD concentrations (fig.~\ref{fig:NNMT_comp_advantage}C and~D). This effect is dependent on a high affinity of NamPRT, as at low affinity (high $K_{M}$), the NADA containing system is again able to maintain higher NAD concentrations. The NAD consumption flux is still higher in the NamPRT and NNMT containing compartment. Taken together this might explain why upon the development of a sufficiently high affinity of NamPRT for its substrate, which seems to have been induced by the appearance of NNMT, NADA is lost.
