%%% TeX-master: "manuscript"
% !TeX spellcheck=en_GB

\section{Results}

\subsection{Paradoxical evolutionary correlation between NAD-dependent signaling and precursor metabolism}

Two principal pathways exist that recycle Nam. As shown in figure \ref{fig:pathway_overview}, the major pathway found in most prokaryotes, yeast, and plants includes four steps starting with the deamidation of Nam to nicotinic acid by NADA. The other three enzymes comprise the Preiss-Handler pathway that also exists in vertebrates. The recycling pathway found in mammals directly converts Nam into Nam mononucleotide (NMN) a reaction catalysed by NamPRT and driven by a non-stoichiometric ATP-hydrolysis. A similar reaction catalysed by an evolutionary related enzyme, NAPRT, converts NA into the corresponding mononucleotide (NAMN) in the Preiss-Handler pathway. NMN and NAMN are converted into dinucleotides by Nam/NA adenylytransferases (NMNATs). The recycling pathway via NA finally requires re-amidation of the dinucleotide, which is catalysed by NAD synthase. This reaction includes an enzyme adenylation step that consumes ATP. Even though Nam recycling by NADA is energetically less efficient, this pathway is preferentially used by most bacteria, fungi and plants, whereas most metazoans recycle Nam using NamPRT (fig.~\ref{fig:phylo_distribution}A).

Analysing the phylogeny of the NAD-recycling enzymes in Metazoa in more detail revealed that not only NamPRT replaced NADA, but in most organisms, especially in Deuterostomes, NamPRT is found together with NNMT (fig.~\ref{fig:phylo_distribution}B). NNMT methylates Nam to methyl-Nam that is in mammals excreted with the urine, thus removing Nam from recycling. NNMT seems to have arisen \textit{de~novo} in the common ancestor of Ecdysozoa and Lophotrochozoa. We were unable to find any \textit{NNMT} gene with considerable\todo{define considerable} similarity in fungi or plants. Even though fungal genes named \textit{NNMT} can be found in databases, these genes show very limited sequence similarities to the \textit{NNMT} of nematodes or deuterostomes. The yeast NNMT protein has meanwhile been shown to be a lysin-protein-methyltransferase \cite{Wlodarski2011}.

Nematodes are the only organisms, where we observed any combination of NADA, NamPRT, and NNMT. In Deuterostomes, the only large clade that possesses only NamPRT and seems to have lost NNMT again are Sauropsida and among them especially birds. The reason why a lot of\todo{be more precise!} bird genomes do not seem to encode NNMT remains unclear, as the distribution is quite scattered (suppl. fig.~S2). The lack of NNMT might be related to the excretion system, as the product of NNMT, methyl-Nam, is in mammals excreted with the urine. There are some Metazoan species for which we could not find NamPRT or NADA, while NNMT was present. We assume that this is due to incomplete genomes in the database, as the distribution of such species is scarce and widely scattered.

In addition to the phylogenetic distribution of the two Nam salvage enzymes NADA and NamPRT, we analysed the phylogenetic diversity of enzymes catalysing NAD-dependent signalling reactions. To do so, we used the previously established classification into ten different families of NAD-consuming signalling enzymes \cite{Gossmann2012FEBS}. The detailed list of templates used for the phylogenetic analyses can be found in supplementary table~S1. The numbers shown in figure~\ref{fig:phylo_distribution}B denote the average number of NAD-dependent signalling enzyme families found in each clade. With the exception of Cnidaria and Lophotrochozoa, we find an average of three to four families in Protostomes, whereas most Deuterostome species have, on average, more than eight families with an increasing diversification of enzymes within some of these families \cite{Gossmann2014DNAR}.

Taken together, we found that NADA is lost in vertebrates, but strongly preserved in most other organisms, despite the higher energetic requirement of that pathway. Moreover, the selection for having both NamPRT and NNMT coincides with an increased diversification of NAD-dependent signalling. This observation seems counterintuitive. One would expect that increased NAD-dependent signalling should be accompanied by an increase of substrate availability for NAD biosynthesis. However, NNMT irreversibly removes Nam from salvage back to NAD.


\subsection{Functional properties of NamPRT and NNMT have evolved to maximize NAD-dependent signaling}

To resolve this apparent contradiction, we wished to scrutinize the NAD metabolic network. Given the complexity of this network, we turned to modelling approaches and built a dynamic model of NAD metabolism based on previously reported kinetic data (for details, see experimental procedures and suppl. tab.~S2).

To be able to compare metabolic features of evolutionary quite different systems in our simulations and as we have limited information about expression levels of enzymes or changes of kinetic constants during evolution, we initially used the kinetic constants found for human or yeast enzymes for all systems analysed and used equal amounts of enzymes for all reactions\todo{Shorten or split sentence and correct "equal amounts"}. Wherever possible, we did not only include substrate affinities but also known product inhibitions or inhibition by downstream metabolites. As we assume that cell growth is, besides NAD-consuming reactions, a major driving force for NAD biosynthesis, we analysed different growth rates (cell division rates) by simulating different dilution rates for all metabolites.

First, we addressed the unexpected correlation between the selection for co-occurrence of NamPRT and NNMT and an increase in the number of NAD-consuming enzyme families. We calculated steady state NAD concentrations and NAD consumption rates by simulating NAD biosynthesis proceeding via NamPRT in the presence or absence of NNMT. As shown in figure~\ref{fig:NNMT_NAD_flux}, the presence of NNMT enables higher NAD consumption rates (fig.~\ref{fig:NNMT_NAD_flux}A) while reducing the steady state concentration of NAD (fig.~\ref{fig:NNMT_NAD_flux}B). The decline in NAD concentration can be compensated by a higher expression of NamPRT, further increasing NAD consumption flux (dashed lines in fig.~\ref{fig:NNMT_NAD_flux}A and~B).

These results can be explained when looking in more detail at the kinetic parameters of NamPRT and NAD-consuming enzymes such as Sirtuin~1. Most NAD-consuming enzymes are inhibited by their product Nam, explaining why the presence of NNMT enables higher NAD consumption fluxes. At the same time, the high substrate affinity of NamPRT maintains a sufficiently high NAD concentration, although the concentration is lower than in the system without NNMT.

As measured kinetic parameters of NamPRT are only available for the human enzyme \cite{Burgos2008}, we analysed the potential effect of NamPRT affinity ($K_{M}$) on NAD steady state concentration and NAD consumption flux. In the absence of NNMT, a change in $K_{M}$ has very little effect on steady state NAD concentration and NAD consumption flux (fig.~\ref{fig:NamPRT_affinity_Nam}A and~B). In the presence of NNMT, however, NAD consumption flux and NAD concentration increases with decreasing $K_{M}$ values (fig.~\ref{fig:NamPRT_affinity_Nam}C and~D).

Remarkably, NAD concentration and consumption flux are both considerably affected by cell division rates in a system without NNMT, at least if the enzyme expression is kept constant at different cell division rates. Of course, this is an artificial scenario, as one would assume organisms to regulate enzyme expression to achieve similar levels of metabolite concentrations instead. However, there seems to be a tradeoff between maintainable NAD concentration and consumption flux in the absence of NNMT. In contrast, in the presence of NNMT, NAD consumption rates and concentrations are almost independent of cell division rates.

Figure~\ref{fig:NamPRT_affinity_Nam}E and~F shows a direct comparison of different affinities of NamPRT for Nam with and without NNMT. At an affinity of $K_{M}$ = 1\,$\mu$M, which is in the range of the $K_{M}$ of NADA for Nam, NAD consumption flux is higher with NNMT than without, except for low cell division rates (fig.~\ref{fig:NamPRT_affinity_Nam}E). At a lower affinity of 5\,nM, increase in consumption rate even more pronounced. The NAD concentration is lower with NNMT than without for both tested $K_{M}$ values (fig.~\ref{fig:NamPRT_affinity_Nam}F).

As pathway dynamics are not solely dependent on one enzyme, we wanted to estimate the impact of the substrate affinity of NNMT that is competing with NamPRT for the same substrate. In figure~\ref{fig:optimal_substr_affinities} is shown that the substrate affinity values found in the human enzymes (indicated by black asterisks) are actually optimal with respect to both achievable steady state NAD concentration and consumption fluxes. Thus, a further increase of the affinity of NamPRT for Nam or a change in the affinity of NNMT for Nam would not provide any advantage.


\subsection{Sequence variance acquired in metazoans enhances substrate affinity}

As our simulations suggested that NNMT might have exerted an evolutionary pressure on the development of NamPRT, we created a multiple sequence alignment to see if we can find sequence variations in NamPRT upon the appearance of NNMT. The multiple sequence alignment of selected sequences is shown in figure~\ref{fig:unresolved_loop}A and a more comprehensive multiple sequence alignment containing a larger number of species can be found in supplementary figure~S1. We found that most Deuterostomes that possess only NamPRT and NNMT (indicated by the blue circle) have an insert of ten amino acids corresponding to positions 42 to 51 in the human enzyme. This insert overlaps with a predicted weak nuclear localisation signal (NLS), that is lost when the insert is removed. The sequence corresponds to a loop at the protein surface that is unresolved in all available crystal structures of human NamPRT (e.g. structure visualisation in fig.~\ref{fig:unresolved_loop}B from \cite{Wang2006}). The loop is connected to one of the $\beta$-sheets involved in substrate binding.

From these observations, we derived two possible hypotheses regarding the role of the loop in NamPRT function. The first hypothesis was that the presence of the loop changes the subcellular localisation of NamPRT, since the loop is part of an NLS. To test this hypothesis, we recombinatly expressed wildtype NamPRT and a mutated version lacking amino acids 42 to 51. Both, wildtype and mutant, were modified to carry a FLAG-tag for easier visualisation. Immunofluorescence imaging showed a mixed cytosolic nuclear localisation for the wildtype and the mutant NamPRT (fig.~\ref{fig:unresolved_loop}C). The presence of the loop does not seem to have any effect on the subcellular localisation.

The second hypothesis was that the sequence insertion influences the affinity of NamPRT, which would be in line with the predictions from our simulations. We recombinantly expressed both wildtype and mutant NamPRT in \textit{E.~coli}. After purification (suppl. fig.~S3), we measured the activity using NMR detection of NMN in the presence and absence of ATP. The enzymatic activity of the mutant enzyme lacking the loop was much lower than the activity of the wildtype enzyme (fig.~\ref{fig:unresolved_loop}D). An increase of the substrate concentration had a much stronger effect on the activity of the mutant compared to the wildtype enzyme, indicating a lower affinity. In contrast to wildtype NamPRT, the mutant does not exhibit an increased activity in the presence of ATP (fig.~\ref{fig:unresolved_loop}E).


\subsection{NamPRT and NNMT make NADA obsolete in vertebrates}

A remaining puzzle was, why NADA is lost in vertebrates, while NADA and NamPRT both occur in bacteria. We tried to approach the puzzle with a competitive model with two compartments that share a Nam source. One compartment contains NADA while the other contains either NamPRT alone, or together with NNMT (for details, see experimental procedures and suppl. tab.~S2). Without NNMT, the compartment containing NADA shows slightly lower NAD consumption rates (fig.~\ref{fig:NNMT_comp_advantage}A), but is able to maintain much higher NAD concentrations at low cell division rates (fig.~\ref{fig:NNMT_comp_advantage}B). At high cell division rates, steady state concentrations in both compartments are similar. These comparable concentrations might explain why in bacteria that often have relatively high growth rates, both systems co-exist.

In the presence of NNMT, the NamPRT compartment has both higher NAD consumption rates and higher steady state NAD concentrations than the compartment containing NADA (fig.~\ref{fig:NNMT_comp_advantage}C and~D). The high NAD concentrations are dependent on a high affinity of NamPRT. When simulating a low affinity (high $K_{M}$) of NamPRT, the NADA compartment is able to maintain higher NAD concentrations although it still has a lower the NAD consumption flux. Taken together, the results suggest that the NADA pathway became less worthwhile when a higher affinity of NamPRT developed, possibly induced by NNMT.
